### DISCUSSION TITLE
Does default compilation embed NNUE weights in the binary

### DISCUSSION THREAD

**Discussion by jxu** [official-stockfish/Stockfish#4173] [Q&A] [answered]
I have the binary generated from
```
    cd src
    make help
    make net
    make build ARCH=x86-64-modern
```
and I copied it to /usr/local, so nnue file is not nearby. Are the weights embedded in the binary?

**Comment by mstembera** [comment, marked as answer]
yes

**Reply by jxu** [reply]
yes I saw in the log file it said "info string NNUE evaluation using nn-6877cd24400e.nnue enabled"
